app/services/prison_visits/client.rb

```diff
@@ -52,11 +53,10 @@ def request(method, route, params)
         }
       }.deep_merge(params_options(method, params))
 
-      Rails.logger.info do
-        "Calling PVB API: #{method.to_s.upcase} #{path}"
-      end
-
-      response = @connection.request(options)
+      message = "Calling PVB API: #{api_method}"
+      response = Instrumentation.time_and_log(message, :api) {
```

Comment:
This block will record the total time spent for all retries. In fact the `time_and_log` message is designed to be called multiple times with the same key, and when doing so you get the sum of all the times. It's semantically the same as total db or total view time in the request log.